testing/upload_validation.ts

```diff
@@ -192,7 +192,11 @@ export function validateSyncTableSchema(schema: any): ArraySchema<ObjectSchema<a
 function getNonUniqueElements<T>(items: T[]): T[] {
```

Comment:
I got too cute with the typing here, I think we only use it for strings, so it's also fine to just type this as `item: string[]` or do `<T extends string>` as the generic (handles strings and enums).